<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication1Class</class>
 <widget class="QMainWindow" name="QtGuiApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48, 118, 172)</string>
   </property>
   <widget class="QPushButton" name="hotButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/Resources/hot.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="starButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/Resources/star.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="hotScrollArea">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>960</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1000</width>
       <height>10000</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>10000</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/Resources/search.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="searchScrollArea">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>960</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1000</width>
       <height>10000</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>10000</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="starScrollArea">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>960</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1000</width>
       <height>10000</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>10000</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="weiboButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/Resources/weiboButton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dongmanButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/Resources/dongmanButton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="qidianButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/Resources/qidianButton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shipinButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/Resources/shipinButton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="zhihuButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/Resources/zhihuButton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>zhihuButton</zorder>
   <zorder>shipinButton</zorder>
   <zorder>qidianButton</zorder>
   <zorder>dongmanButton</zorder>
   <zorder>weiboButton</zorder>
   <zorder>hotButton</zorder>
   <zorder>starButton</zorder>
   <zorder>searchButton</zorder>
   <zorder>hotScrollArea</zorder>
   <zorder>searchScrollArea</zorder>
   <zorder>starScrollArea</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>searchClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hotButton</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>hotClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>-6</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>starButton</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>starClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>-6</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weiboButton</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>weiboClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>-18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dongmanButton</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>dongmanClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qidianButton</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>qidianClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zhihuButton</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>zhihuClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>-4</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shipinButton</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>shipinClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>-6</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>button()</slot>
  <slot>searchClicked()</slot>
  <slot>hotClicked()</slot>
  <slot>starClicked()</slot>
  <slot>weiboClicked()</slot>
  <slot>slot1()</slot>
  <slot>dongmanClicked()</slot>
  <slot>qidianClicked()</slot>
  <slot>zhihuClicked()</slot>
  <slot>shipinClicked()</slot>
 </slots>
</ui>
